### PR TITLE
Update from bootstrap version 

### PR DESCRIPTION
**Pull request by hniyazi**

### CODE REVIEW

File: dev/_sass/_bootstrap/_utilities.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@import "../../node_modules/bootstrap/scss/utilities/display";
+@import "../../node_modules/bootstrap/scss/utilities/embed";
+@import "../../node_modules/bootstrap/scss/utilities/flex";
+@import "../../node_modules/bootstrap/scss/utilities/float";
+@import "../../node_modules/bootstrap/scss/utilities/position";
+@import "../../node_modules/bootstrap/scss/utilities/screenreaders";
+@import "../../node_modules/bootstrap/scss/utilities/shadows";
+@import "../../node_modules/bootstrap/scss/utilities/sizing";
+@import "../../node_modules/bootstrap/scss/utilities/spacing";
+@import "text";
```
Comment (nijat13) [inline dev/_sass/_bootstrap/_utilities.scss:14]: Why text, not `../../node_modules/bootstrap/scss/utilities/text`

File: dev/_sass/_bootstrapcomponents/_cards.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//Cards
+.card {
+
+  // Card example
```
Comment (nijat13) [inline dev/_sass/_bootstrapcomponents/_cards.scss]: Remove comments

File: dev/_sass/_project-ui.scss [hunk 3]
```diff
@@ -66,14 +66,6 @@
...
 }
 
-.section {
-  margin-bottom: $section-distance !important;
-}
-
-.img-responsive-inline {
-  @include img-responsive(inline-block);
-}
-
```
Comment (nijat13) [inline dev/_sass/_project-ui.scss:76]: Why those styles have been removed?

Comment (hniyazi) [reply dev/_sass/_project-ui.scss:76]: This mixin included from bootstrap v3 mixins

File: dev/_sass/_bootstrapcomponents/_cards.scss (hunk 2 above)
Comment (hniyazi) [reply dev/_sass/_bootstrapcomponents/_cards.scss]: Done!

File: dev/_sass/_bootstrap/_utilities.scss (hunk 1 above)
Comment (hniyazi) [reply dev/_sass/_bootstrap/_utilities.scss:14]: In text.scss file I removed `@include text-hide();` cause it made an error

Comment (nijat13) [reply dev/_sass/_bootstrap/_utilities.scss:14]: Oh got it!

File: dev/_sass/_project-ui.scss (hunk 3 above)
Comment (nijat13) [reply dev/_sass/_project-ui.scss:76]: Doesn't this className usable anymore?
Or you've moved to another place?

Comment (hniyazi) [reply dev/_sass/_project-ui.scss:76]: Bootstrap v4 has new mixin `img-fluid`

Comment (nijat13) [reply dev/_sass/_project-ui.scss:76]: Got it!

**Review by nijat13 — APPROVED** [review]

**Pull request merged**